src/modules/integrator-guide/pages/security-model/permissions-evaluation.adoc

```diff
@@ -0,0 +1,313 @@
+include::ROOT:partial$variables.adoc[]
+
+= Permission Evaluation
+
+The permission evaluation feature allows administrators to check which actions or operations can be performed by a user or a combination of roles.
+The evaluation entails comparing the different roles assigned to a user to determine the user's ability to perform the various operations within a resource.
+
+[NOTE]
+====
+xref:access-control/index.adoc[{PRODUCT_NAME} RBAC] uses the roles of the given user to determine if they have access to a given resource.
+====
+
+== How to evaluate applied permissions
+
+[NOTE]
+====
+Most of the {PRODUCT_NAME} resources allow you to define which xref:access-control/roles.adoc[roles] are allowed to access the resource.
+Locate the btn:[permissions] button in the user interface in order to manage the resource.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/general-permissions-button.png",
+    "alias": "general-permissions-button",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 320,
+    "y": 0,
+    "h": 550,
+    "w": 1600
+  },
+  "focus": {
+    "x": 1466,
+    "y": 80,
+    "h": 31,
+    "w": 206
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1536,
+    "y": 80,
+    "w": 134,
+    "h": 30
+  }]
+}
+----
+====
+
+Navigate to the desired resource, and click the btn:[permissions].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/permission-button-page.png",
+    "alias": "new-user",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 1535,
+    "y": 75,
+    "w": 130,
+    "h": 42
+  }]
+}
+----
+
+Click the btn:[add +] button.
+A popup window with two dropdown options displays, the btn:[select role(s)] and btn:[search or select user].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/add-role.png",
+    "alias": "add-role",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "padding": "xs",
+    "x": 765,
+    "y": 150,
+    "w": 750,
+    "h": 80
+}]
+}
+----
+
+Click on the btn:[select roles] or the btn:[select user] dropdown and select a role or a user to evaluate.
+Only one of two options can be applied.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/eval-permission.png",
+    "alias": "eval-permission.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "focus": {
```

Comment:
The focus is not ok:

![image](https://user-images.githubusercontent.com/12545711/198608059-6922c257-7646-4d28-a7bd-9c6be7436eac.png)


How I do focus for popups; I firstly align the red box to the popup like so:

![image](https://user-images.githubusercontent.com/12545711/198607291-4b616edf-110c-4e6e-8ce6-933a40fed059.png)

Then I shift + arrow up on the X axis, shift + arrow up on the Y axis; 2x shift + arrow left on the width, 2x shift + arrow down on the height to get it like so

![image](https://user-images.githubusercontent.com/12545711/198607613-dc442b74-7b1b-43cc-910f-e31b318c97fc.png)


Then I use those numbers with `xs` padding to get this:

![image](https://user-images.githubusercontent.com/12545711/198609306-dbfbb615-ebe7-483b-bb6f-b58abe6430fa.png)

Sometimes I'll make it even more into the popup to get rid of those sharp edges, but won't fight you over them.

Reply:
!! Note that I've only fixed the focus, not the annotation